### DISCUSSION TITLE
10.5.0

### DISCUSSION THREAD

**Discussion by SubJunk** [UniversalMediaServer/UniversalMediaServer#2489] [General]
General:
		Added new option to mark a fully played video after moving it to a new folder
		Implemented external API to allow users to perform actions on their UMS instance from an authorized external source, please see docs at https://support.universalmediaserver.com/books/configuration/page/external-api (thanks, ikrahne!)
		Fixed TV series metadata not saving for some series
		Improved episode detection from filenames
		Optimized network use and handling of unimplemented UPnP actions for some renderers
		Fixed audio cover art extraction (thanks, ik666!)
		Fixed database upgrade for some users
		Fixed recognition of some renderers
	Renderers:
		Improved support for H.264 on Panasonic VT60
	Translation updates via Crowdin:
		Danish (99%) (thanks, NCAA and GurliGebis!)
		Finnish (99%) (thanks, Esko Gardner!)
		French (100%) (thanks, Archaos!)
		Italian (98%) (thanks, tiwi90!)
		Korean (99%) (thanks, VenusGirl!)
		Polish (100%) (thanks, Karol Szastok!)
		Portuguese (99%) (thanks, mariopinto!)
		Portuguese (Brazilian) (96%) (thanks, Mauro.A!)
		Serbian (97%) (thanks, Slobodan Simić (Слободан Симић)!)
		Spanish (100%) (thanks, Gerardo Ruiz, fafranco82, Eduardo Martin, and manuel fernandez!)
		Turkish (100%) (thanks, Burak Yavuz!)
		Ukrainian (17%) (thanks, Paul Furlet!)
	Dependencies:
		Bump checkstyle from 8.41 to 8.42
		Bump com.sun.xml.bind-version from 3.0.0 to 3.0.1
		Bump commons-lang3 from 3.11 to 3.12.0
		Bump icu4j from 68.2 to 69.1
		Bump JRE from 15 to 8u292
		Bump maven-project-info-reports-plugin from 3.1.1 to 3.1.2
		Bump MediaInfo from 20.09 to 21.03
		Bump metadata-extractor from 2.15.0 to 2.16.0
		Bump spotbugs-maven-plugin from 4.2.2 to 4.2.3
		Bump twelvemonkeys-imageio-version from 3.6.4 to 3.7.0

<hr /><em>This discussion was created from the release <a href='https://github.com/UniversalMediaServer/UniversalMediaServer/releases/tag/10.5.0'>10.5.0</a>.</em>

**Comment by DeFlanko** [comment]
Big Plays! Nice to see your back on the Project! What's coming down the pipe?

**Reply by SubJunk** [reply]
@DeFlanko hey man :) good to see you. We have some pretty big things on the way. Probably the biggest new thing is cloud media access so that people can access their media via a web interface, even if their computer is switched off. And then it opens the door to making it much easier to have more and better features since that stuff is built on a much more modern toolset.

It's a really big chunk of work as you can imagine! Also the balance between fixing bugs as usual, and trying to improve documentation... We have a new knowledge base that I hope can become the one place for documentation, so that needs some work to migrate the existing docs from all their places.

What have you been working on lately?

**Comment by DeFlanko** [comment]
Mostly HL7 Interfaces and integrations, still doing a lot of SQL reporting.
I might be able to pitch in some time in the documentation part -- is a lot
of it just in the code as comments right now?

On Mon, Jul 12, 2021 at 2:00 PM SubJunk ***@***.***> wrote:

> @DeFlanko <https://github.com/DeFlanko> hey man :) good to see you. We
> have some pretty big things on the way. Probably the biggest new thing is
> cloud media access so that people can access their media via a web
> interface, even if their computer is switched off. And then it opens the
> door to making it much easier to have more and better features since that
> stuff is built on a much more modern toolset.
>
> It's a really big chunk of work as you can imagine! Also the balance
> between fixing bugs as usual, and trying to improve documentation... We
> have a new knowledge base that I hope can become the one place for
> documentation, so that needs some work to migrate the existing docs from
> all their places.
>
> What have you been working on lately?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/UniversalMediaServer/UniversalMediaServer/discussions/2489#discussioncomment-995790>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABH52MKESBM6IS4JUJPKP2DTXNJV3ANCNFSM44OGURHA>
> .
>


-- 
Thanks, James

**Reply by SubJunk** [reply]
Oh that’s important work, nice one

Would be great to get help on that! It’s more about getting our user-targeted docs out of the Help tab in the program and the wiki, and into support.universalmediaserver.com, so there is just one place for users to look for information.

Then I hope to replace that Help tab with a link to that website, and have the wiki just for devs
